get_github_coderabbit_review/coderabbit_fetcher/dependency_injection/container.py

```
@@ -0,0 +1,557 @@
+        for param_name, param in signature.parameters.items():
+            if param_name == "self":
+                continue
+
+            # Skip parameters with default values if no binding exists
+            if param.annotation == param.empty:
+                if param.default != param.empty:
+                    continue
+                else:
+                    raise DIError(
+                        f"Parameter {param_name} in {implementation_type.__name__} has no type annotation"
+                    )
```

Ruff (0.13.2)

Avoid specifying long messages outside the exception class

(TRY003)